src/App.js

```diff
@@ -6,15 +6,17 @@ import Navbar from './components/Navbar';
 import './App.css';
 
 const App = () => (
-  <Router>
-    <Navbar />
-    <div>
-      <Routes>
-        <Route path="/books" element={<Books />} exact />
-        <Route path="/categories" element={<Categories />} exact />
-      </Routes>
-    </div>
-  </Router>
+  <div className="panel-bg">
+    <Router>
+      <Navbar />
+      <div>
+        <Routes>
+          <Route path="/books" element={<Books />} exact />
+          <Route path="/categories" element={<Categories />} exact />
+        </Routes>
```

Comment:
@elyor-doniyorov Thank you for your feedback. Can you please be more specific and provide me with some feedback, because I tried many times but I don't have this bug when I try on my local machine.